frontend/src/metabase/ui/components/inputs/MultiAutocomplete/MultiAutocomplete.unit.spec.tsx

```diff
@@ -140,15 +140,15 @@ describe("MultiAutocomplete", () => {
 
   it("should accept quote-delimited values containing commas when pasting", async () => {
     const { input, onChange } = setup({ data: EXAMPLE_DATA });
-    input.focus();
+    act(() => input.focus());
```

Comment:
calling `userEvent.click()` might be an alternative here that is more "what a user would do".